### ISSUE TITLE
Improve error handling when .env file is not present

### ISSUE THREAD

**Issue by azrikahar** [directus/examples#71]
Need to verify this for all existing examples except SvelteKit (which was implemented in #70).

**Comment by freekrai** [comment]
@azrikahar I have a few ideas about this, will vary based on stack of course but zod is useful for checking env vars

**Comment by azrikahar** [comment]
Sounds great! Zod definitely would be more robust here 👍

**Comment by freekrai** [comment]
Yeah zod or envalid, where envalid is specifically for env vars, so will pick one.

The fresh example currently uses envalid but once I choose one I'll update them all to be the same.

**Comment by rijkvanzanten** [comment]
Linear: [ENG-100](https://linear.app/directus/issue/ENG-100/improve-error-handling-when-env-file-is-not-present)
